src/audio/tdfb/tdfb.c

```diff
@@ -369,11 +561,21 @@ static int tdfb_cmd(struct comp_dev *dev, int cmd, void *data,
 
 	switch (cmd) {
 	case COMP_CMD_SET_DATA:
+		//comp_info(dev, "tdfb_cmd(): COMP_CMD_SET_DATA");
```

Comment:
curious why these are commented out

Reply:
Too verbose? I used those traces to see what type of new commands need to be handled.

Reply:
Why not make dbg level then? just trying to avoid commented code without explanation :)